packaging/os/pkgutil.py

```diff
@@ -63,10 +63,10 @@
 import pipes
 
 def package_installed(module, name):
-    cmd = [module.get_bin_path('pkginfo', True)]
+    cmd = ['pkginfo']
```

Comment:
this is not a good change, get_bin_path makes sure it finds a pkginfo or returns a module fatal error, now you assume the pkginfo is in path and the run_command must deal with it not being there